packages/plugins/rubygem-foreman_kubevirt/rubygem-foreman_kubevirt.spec

```
@@ -0,0 +1,109 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_kubevirt
+%global plugin_name kubevirt
+%global foreman_min_version FIXME
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.0
+Release: 1%{?foremandist}%{?dist}
+Summary: Provision and manage Kubevirt Virtual Machines from Foreman
+Group: Applications/Systems
+License: GPL-3.0
```

RPM packaging uses different codes that predate SPDX

```suggestion
License: GPLv3
```

---

fixed